vite.config.js

```diff
@@ -22,6 +22,6 @@ export default defineConfig({
     emptyOutDir: true,
   },
   define: {
-    'process.env': process.env
+    "process.env": process.env, // temporary workaround for parse-link-header process.env issue
```

Comment:
```suggestion
    "process.env": process.env, // workaround for parse-link-header library that depends on 2 vars defined in `process.env`, see https://github.com/thlorenz/parse-link-header/issues/31
```

Reply:
The motivation for the comment is to be able to easily check that the issue was solved. If there is no precise description or a link here, nobody will check it later.